src/containers/PipelineRuns/PipelineRuns.test.js

```diff
@@ -0,0 +1,285 @@
+/*
```

Comment:
Would tests be possible for some of the problem cases found during the review

Reply:
I've added a test to check duplicate filters are prevented and to check that an invalid filter such as `foo=bar` is also reported, are there any others you would suggest?